packages/dropdown/src/Dropdown.tsx

```diff
@@ -265,22 +265,24 @@ export const Dropdown = React.forwardRef(
           },
         })}
         after={
-          <DropdownList
```

Comment:
Burde dette kanskje heller håndteres inne i DropdownList? At den kan ta in readOnly / disabled og i de tilfellene returnere `null`